components/CodeSnippet.vue

```diff
@@ -64,20 +75,109 @@ const copied = ref(false);
 
 defineExpose({ modalVisible });
 
-const parameters = computed(() => appStore.currentScenario.parameters);
+const everyScenarioHasParameters = computed(() => props.scenarios.every(scenario => !!scenario.parameters));
+
+// TODO: (jidea-317) In the future, we should have the R API capture the actual call made to the model,
+// returning it as part of the result response, so that it can be displayed here verbatim.
+// In that way, we'd avoid two things:
+// (1) replicating logic or constants from the R API package here,
+// (2) the risk of the code snippet generation being out of date with respect to the current model interface.
+
+// Values copied from https://github.com/jameel-institute/daedalus.api/blob/main/R/behaviour.R
+const OPTIMISM = Object.freeze({
```

Comment:
Could you pop in a comment that this is reversed (words to values) because the option title relates to change in behaviour, which is linked to risk perception? The link to {daedalus.api} is liable to be brittle.